tests/posix/common/src/pthread.c

```diff
@@ -233,6 +233,28 @@ void *thread_top_term(void *p1)
 	return NULL;
 }
 
+ZTEST(posix_apis, test_pthread_condattr)
```

Comment:
If you end up having to respin this for whatever reason, this should probably go into cond.c